.github/workflows/xtest.yml

```diff
@@ -241,20 +256,24 @@ jobs:
       - uses: bufbuild/buf-setup-action@2211e06e8cf26d628cda2eea15c95f8c42b080b3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Set up JDK
+        if: matrix.sdk == 'java'
```

Comment:
'focus=x' means 'run all tests that involve sdk x', not 'run tests that involve ONLY sdk x' - so you'll still need the other SDKs for the cross-tests. If you want to save time building the sdks, you can either build in a jobs and download them in the test job, and/or use published artifacts (which we'd have to start doing somehow).